awx/ui_next/src/components/PaginatedDataList/PaginatedDataList.jsx

```diff
@@ -111,76 +66,61 @@ class PaginatedDataList extends React.Component {
       i18n,
       renderToolbar,
     } = this.props;
-    const columns = toolbarColumns.length
-      ? toolbarColumns
-      : [{ name: i18n._(t`Name`), key: 'name', isSortable: true }];
-    const queryParams = parseNamespacedQueryString(qsConfig, location.search);
+    const columns = toolbarColumns.length ? toolbarColumns : [{ name: i18n._(t`Name`), key: 'name', isSortable: true, isSearchable: true }];
+    const queryParams = parseQueryString(qsConfig, location.search);
 
     const itemDisplayName = ucFirst(pluralize(itemName));
-    const itemDisplayNamePlural = ucFirst(
-      itemNamePlural || pluralize(itemName)
-    );
+    const itemDisplayNamePlural = ucFirst(itemNamePlural || pluralize(itemName));
 
     const dataListLabel = i18n._(t`${itemDisplayName} List`);
-    const emptyContentMessage = i18n._(
-      t`Please add ${itemDisplayNamePlural} to populate this list `
-    );
+    const emptyContentMessage = i18n._(t`Please add ${itemDisplayNamePlural} to populate this list `);
     const emptyContentTitle = i18n._(t`No ${itemDisplayNamePlural} Found `);
 
     let Content;
     if (hasContentLoading && items.length <= 0) {
-      Content = <ContentLoading />;
-    } else if (contentError) {
-      Content = <ContentError error={contentError} />;
```

Comment:
We want to keep passing the whole `contentError` object to the `ContentError` components here - this looks like it might be an unintentional change from resolving this PR with recent changes to error handling upstream. The content error components now show expandable error details, so they expect the whole error object now.